manifests/install.pp

```diff
@@ -81,8 +81,8 @@
 
   $packages =[
     gcc,
-    python36,
-    python36-devel,
+    python39,
+    python39-devel,
```

Comment:
Most distros don't have packages named `python39` or `python39-devel`, at least not available in their main repos.  The only match I could find is in CentOS 8 AppStream.  It would be best if these could be overridden.  For example, in CentOS 7, the SCL package is named `rh-python38-python`.

Reply:
This module supports CentOS and RedHat 8 only... So I bump this version to match with NetBox v3 requirements (with main repositories of these distros). If you want to make this module compatible with CentOS 7, indeed you must have as a prerequisite the SCL repository and specify the version 3.8. I don't think so this feature should be in this PR...

Reply:
Ok, I filed https://github.com/anderssh/ash-netbox/issues/24